### ISSUE TITLE
Upgrade to Symfony 3

### ISSUE THREAD

**Issue by jurgenhaas** [acquia/acquia-sdk-php#78]
Are you planning to upgrade this SDK to later versions eventually? I'd like to integrate this into my DRD module (https://www.drupal.org/project/drd) which runs on Drupal 8.4 and there I can't install this SDK which requires Symfony ~2

**Comment by skwashd** [comment]
@jurgenhaas [based on the recent update to the README.md](https://github.com/acquia/acquia-sdk-php/commit/88fb1f7786a8a9256e7e5f1b7ac2aaf3aa679483) I don't think this will happen.

**Comment by jurgenhaas** [comment]
Thanks @skwashd for pointing me that direction. I still find it kind of weired for two reasons:

1) This SDK lives in the Acquia namespace over at [packagist](https://packagist.org/packages/acquia/acquia-sdk-php)
2) There doesn't seem to be any alternative library to talk to their API

Well, for my own use case with DRD I've solved it directly as I had only a couple of endpoints to talk to and that was pretty simple. I'm just a bit confused that a company like Acquia doesn't have better API support.

**Comment by skwashd** [comment]
@jurgenhaas it is disappointing to see how Acquia have handled this. Back in 2013 [Acquia were actively promoting the SDK](https://dev.acquia.com/blog/using-composer-manager-get-island-now) as well as encouraging customers to use it.

I decided to write a [Python library for interacting with the Cloud API](https://github.com/skwashd/python-acquia-cloud). Given what has happened I'm very relieved that I fought internally to use Python instead of PHP. The code is still used daily to manage Drupal deployments at scale.

**Comment by aaronpacheco** [comment]
I agree that it is unfortunate for this SDK to no longer be supported -- I know a number of people have come to rely on it, or were otherwise hoping to build off of it.

For additional context, the Acquia SDK was maintained as a pet project by a handful of developers at Acquia while they were still with the company, but it never really gained widespread adoption by customers, so it was never formally adopted as an official product or service. That is one of the challenging things with APIs/SDKs -- if customers don't broadly adopt them or explicitly ask for them, they tend not to get high prioritization for improvements or formal adoption in the grand scheme of things.

All I can say is that if more customers provide feedback directly to their Account Managers or our Partner Services teams about the importance of a robust, maintained SDK, or if explicit requests are made for API improvements through those channels, there's a better chance they'll land on an official roadmap at some point. I'd certainly love to see that happen, and I imagine many others would, too.

**Comment by typhonius** [comment]
I've started the process of writing a basic SDK that talks to Cloud API v2 here: https://github.com/typhonius/acquia-php-sdk-v2

I'd appreciate assistance from anyone wishing to enhance the SDK as I'm sure there are plenty of edge cases I haven't catered for.

I'm going to be using this and further developing it for the Acquia Cloud Cli command that we've been using (https://github.com/typhonius/acquia_cli).
